client-templates/github-enterprise/accept.json.sample

```diff
@@ -1311,6 +1311,18 @@
       "path": "/repos/:name/:repo/contents/:path*%2F.snyk",
       "origin": "https://${GITHUB_TOKEN}@${GITHUB_API}"
     },
+    {
+      "//": "used to scan Dockerfile",
+      "method": "GET",
+      "path": "/repos/:name/:repo/contents/:path*/Dockerfile",
+      "origin": "https://${GITHUB_TOKEN}@${GITHUB_API}"
```

Comment:
also seen `RAW` in this accept.json

Reply:
Hmm, I was under the impression GHE shouldn't have access to anything on RAW. I will double check that, but rather than affect this it will probably be a separate PR to remove the unnecessary RAWs if it turns out to be as I expect.

Reply:
So on further digging, it's valid for there to be a RAW URL for GHE and users can configure theirs in the .env file, so I've added these lines for RAW to the GHE whitelist too.